scripts/apps/monitoring/controllers/MonitoringController.js

```diff
@@ -34,24 +44,33 @@ export function MonitoringController($rootScope, $location, desks) {
 
     var vm = this;
 
+    vm.switchView(vm.viewColumn);
+
     function preview(item) {
         vm.previewItem = item;
         vm.state['with-preview'] = !!item;
-        if (vm.previewItem != null){
+
+        if (vm.previewItem != null) {
             vm.showHistoryTab = vm.previewItem.state !== 'ingested';
         }
+
+        if (!item) {
+            vm.selectedGroup = null;
+        }
     }
 
     function closePreview() {
         preview(null);
         $rootScope.$broadcast('item:unselect');
+        if (vm.viewColumn) {
+            $rootScope.$broadcast('resize:header');
```

Comment:
This doesn't seem like something that should be an _app-wide_ event. I would find another way to do this.